### PR TITLE
topbar dialog fix

### PR DESCRIPTION
**Pull request by goldylucks**
Using a fork of https://github.com/ethcore/parity-dapps-builtins-rs until @tomusdrw is availble

### CODE REVIEW

File: dapps/Cargo.toml [hunk 1]
```diff
@@ -23,7 +23,7 @@ ethcore-util = { path = "../util" }
 parity-dapps = { git = "https://github.com/ethcore/parity-dapps-rs.git", version = "0.3" }
 # List of apps
 parity-dapps-status = { git = "https://github.com/ethcore/parity-dapps-status-rs.git", version = "0.5.0" }
-parity-dapps-builtins = { git = "https://github.com/ethcore/parity-dapps-builtins-rs.git", version = "0.5.0" }
+parity-dapps-builtins = { git = "https://github.com/goldylucks/parity-dapps-builtins-rs.git", version = "0.5.2" }
```
Comment (rphmeier) [inline dapps/Cargo.toml]: prefer to have this under the ethcore umbrella

Comment (arkpar) [inline dapps/Cargo.toml]: repo restructuring will happen soon

**Comment by tomusdrw** [comment]
@goldylucks Permissions to ethcore/parity-dapps-builtins-rs granted.

**Comment by gavofyork** [comment]
will wait until it's in the ethcore repo...

**Comment by rphmeier** [comment]
Moved into ethcore repo. I think this is also supposed to be in 1.2.1?

**Pull request merged**